# Generated by Django 5.0.8 on 2024-11-21 04:43

import uuid
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='UnexpectedEvent',
            fields=[
                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
                ('created', models.DateTimeField(auto_now_add=True, verbose_name='Creation date')),
                ('exception_type', models.TextField(blank=True, null=True, verbose_name='Exception Type')),
                ('exception_msg', models.TextField(blank=True, null=True, verbose_name='Exception Msg')),
                ('traceback', models.JSONField(blank=True, null=True)),
                ('detail', models.JSONField(blank=True, null=True)),
                ('item', models.CharField(blank=True, max_length=256, null=True, verbose_name='Item')),
                ('action', models.CharField(blank=True, max_length=256, null=True, verbose_name='Action')),
            ],
            options={
                'ordering': ['-created'],
                'indexes': [models.Index(fields=['exception_type'], name='tracker_une_excepti_47ede4_idx'), models.Index(fields=['item'], name='tracker_une_item_795336_idx'), models.Index(fields=['action'], name='tracker_une_action_55cd4b_idx')],
            },
        ),
    ]
